src/ducks/search/SearchPage.jsx

```diff
@@ -163,7 +165,7 @@ const SearchPage = () => {
       </Header>
       {!searchSufficient ? (
         <Padded>
-          <NarrowContent>
+          <NarrowContent style={contentDesktopStyle}>
```

Comment:
`u-m-auto` ?

Reply:
Nice, I wondered about that but did not bother to search, thanks !